src/content/docs/network-performance-monitoring/advanced/advanced-config.mdx

```diff
@@ -2426,9 +2426,9 @@ By default, flow data containers will collect and process every flow packet they
   </Collapser>
 </CollapserGroup>
 
-## Automatically reloading custom SNMP profiles
+## Automatically reloading custom SNMP profiles [#watch-profile-changes]
 
-By default, the `ktranslate` Docker container must be manually destroyed and rebuilt to incorporate changes to the SNMP profiles in the `mib_profile_dir` path. This is normal behavior in most deployments as the Docker image pulls in the latest profiles available from the public [snmp-profiles repository](https://github.com/kentik/snmp-profiles). In situations where you provide [your custom profiles](/docs/network-performance-monitoring/advanced/snmp-profiles/#private), you can use this setting to enable the container to automatically refresh the underlying configurations and SNMP profiles for the container.
+By default, the `ktranslate` Docker container must be manually destroyed and rebuilt to incorporate changes to the SNMP profiles in the [mib_profile_dir](#global) path. This is normal behavior in most deployments as the Docker image pulls in the latest profiles available from the public [snmp-profiles repository](https://github.com/kentik/snmp-profiles). In situations where you provide [custom profiles](/docs/network-performance-monitoring/advanced/snmp-profiles/#private), you can use the [watch_profile_changes](#global) setting to enable the container to automatically refresh the underlying configurations and SNMP profiles for the container.
```

Comment:
@thezackm this has to include the link to the entire URL. Where are you trying to link to here [mib_profile_dir](#global)?

Reply:
@homelessbirds - correct; the target is [right here](https://github.com/newrelic/docs-website/blob/5003c5f1da867d8be3d0942a3c3226bf7ac207de/src/content/docs/network-performance-monitoring/advanced/advanced-config.mdx?plain=1#L921-L924); similar to how it's done in other areas like [HERE](https://github.com/newrelic/docs-website/blob/5003c5f1da867d8be3d0942a3c3226bf7ac207de/src/content/docs/network-performance-monitoring/advanced/advanced-config.mdx?plain=1#L303). Historically the link works as a short link if the target is on the same page. 

Would you like me to update all of these into the full URL?

Reply:
@homelessbirds - are we good to go on this one? I didn't realize it was still sitting, sorry!